### PR TITLE
initial push to server before git ignore and other things

### PR DESCRIPTION
**Pull request by j2kan**
All commented out now because local development you don't need these security features enabled.

Will have a separate PR to add settings.py to git ignore but I wanted you guys to see these settings on the server.

Will also be setting up 3 private files.

1. secret key
2. email pw
3. database pw

And verify i've set up the database to own accept connection from our app.

### CODE REVIEW

**Review by amyleblond — APPROVED** [review]
cool!

and im assuming you mean that the db will only connect to our app

**Review by smwatts — COMMENTED** [review]
Would you be able to walk us through this in standup? Very interesting!

**Comment by j2kan** [comment]
Added more documentation for you guys to read if interested. #CSDegreeRequired? #qi-ming.

File: taworks/taworks/settings.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'ri_*-e$mdbw&q#h2)epdhxi7(@026fir_f(a77)e)%p9t1+lxs'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (j2kan) [inline taworks/taworks/settings.py]: this needs to be false.

**Review by j2kan — COMMENTED** [review]
push latest too

**Review by smwatts — APPROVED** [review]
LGTM!

**Pull request merged**